Utilities/RelMon/python/directories2html.py

```diff
@@ -94,18 +99,34 @@ def get_page_header(directory=None, standalone=False, additional_header=""):
           'a.black_link:hover {color: #737373}\n'+\
           'a.black_link:visited {color: #333333}\n'+\
           'a.black_link:active {color: #333333}\n'
+    ## fixed first row and first column table
+    
+    wrapper_h = min(thead_h + (70 * len(directory.comparisons)),1800)
+
+  tablestyle = '\n.wrapper { overflow: auto; height: %dpx;} \n'%(wrapper_h)
+  tablestyle += 'table {  position: relative; border-collapse: separate; border-spacing: 0;} \n'
+  tablestyle += 'table { position: relative; border-collapse: separate; border-spacing: 0;} \n'
+  tablestyle += 'table th, table td { width: 50px; padding: 5px; background-color: white;} \n'
+  tablestyle += 'table th { position: sticky; top: 0; z-index: 2; height: %dpx;} \n'%(thead_h)
+  tablestyle += 'table th:nth-child(1) { left:0; z-index:3;} \n'
+  tablestyle += '.sticky-col { position: sticky; background-color: #C9FFD1  ; width: 200px; left:0}\n'
+  tablestyle += '.center_head { position: absolute; top: 50%; left: 50%;} \n'
+  tablestyle += '.vertical_head {top: 60%; -webkit-transform:  translateX(-50%) translateY(-50%) rotate(-90deg); -moz-transform: translateX(-50%) translateY(-50%) rotate(-90deg);} \n'
   javascripts=""
+
   
-  
+
   html='<html>'+\
        '<head>'+\
        '<title>RelMon Summary</title>'+\
+       '<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.4/Chart.js"></script>' + \
        '<link rel="stylesheet" href="%s/style/blueprint/screen.css" type="text/css" media="screen, projection">'%style_location+\
```

Comment:
These three stylesheets look like they were served through http://cms-service-reldqm.web.cern.ch which doesn't seem up anymore. Could you remove them?

Reply:
Oh, I found archived versions!

https://web.archive.org/web/20160325045951/https://cms-service-reldqm.web.cern.ch/cms-service-reldqm/style/blueprint/screen.css

https://web.archive.org/web/20160325045954/https://cms-service-reldqm.web.cern.ch/cms-service-reldqm/style/blueprint/print.css

https://web.archive.org/web/20160325045956/https://cms-service-reldqm.web.cern.ch/cms-service-reldqm/style/blueprint/plugins/fancy-type/screen

Not strictly required, but they could be added back this way.

Reply:
Indeed you are right, most probably we could upload them on the RelMon service github repository directly to have them "under control".

Reply:
(I'd do that)

Reply:
@AdrianoDee remove the `/style/blueprint/` and `/style/blueprint/plugins/` since you have all the stylesheets directly in the `style_location` :)